students/dominic_martin/lesson03/Slicing.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env Python 3
+
+'''All this program needs added is an input to allow
+the user to type an arbitrary sequence.  
+
+It also needs unit tests for each function.
+'''
+s = 
+s = [29, 36, 27, 19, 9, 8, 7, 6, 5, 18, 33, 78, 45, 22, 3]
+
+def firstLast(s):
```

Comment:
Per PEP8 function names should be snake_case - for this as well as all functions in this code.